### ISSUE TITLE
Train and test LeNet5-maxpool on the triangles and squares dataset

### ISSUE THREAD

**Issue by elboyran** [NLeSC/XAI#12]
- [x] Create new script to configure parameters. Use [config_params](https://github.com/NLeSC/XAI/blob/master/Software/MATLAB/SimpleLRPExperiment/config_params.m) as starting code.

- [x] Create script for training on the dataset. Use [train_lenet_babyhapes](https://github.com/NLeSC/XAI/blob/master/Software/MATLAB/SimpleLRPExperiment/test_scripts/train_lenet_babyaishapes.m) as a starting code.

- [x] Modify code to reflect the labeling 0=square, 2= triangle

- [x] Train a model  on training and validation sets for binary data

- [x] Test on the test set for binary data

- [x] Train a model  on training and validation sets for gray data

- [x] Test on the test set for gray data

**Comment by elboyran** [comment]
commit d0470b1d61c4f32e4861309bafeaee0e9cff01f6

**Comment by elboyran** [comment]
Training and testing accuracies: 100%!

**Comment by elboyran** [comment]
Training accuracy (after 20000 iterations) is 91.435%
Testing accuracy: 91.31%

**Comment by elboyran** [comment]
commit b43b26d049eeed6b2ed0587e074fcbbfb5349824
